Makefile

```diff
@@ -212,7 +212,7 @@ images-buildx: ## Build all container images necessary to run the whole operator
 	buildah manifest push --all $(DPU_DAEMON_IMAGE)-manifest docker://$(DPU_DAEMON_IMAGE)
 
 .PHONY: images-push
-images-push: test ## Push all container images necessary to run the whole operator
+images-push: ## Push all container images necessary to run the whole operator
```

Comment:
Some commit conflict here. You did this change in another commit.

Reply:
I'll rebase.